labelbox/client.py

```diff
@@ -49,6 +51,7 @@ def __init__(self,
         Args:
             api_key (str): API key. If None, the key is obtained from the "LABELBOX_API_KEY" environment variable.
             endpoint (str): URL of the Labelbox server to connect to.
+            enable_beta (bool): Indicated whether or not to use beta features
```

Comment:
In the comment I would explain the danger of using beta features.

The name beta scares me as well because it implies that we are seriously considering making them G.A., but I'm not sure if that is the case here.

Reply:
We are going to do a full release of these features. We are only holding off until we can make sure they are working as expected.